.github/workflows/ci.yml

```diff
@@ -40,10 +40,11 @@ jobs:
           sudo apt -y install lua5.1 luarocks
           sudo luarocks install lua_pack
           sudo luarocks install lpeg
+          sudo luarocks install lua-resty-string
```

Comment:
Interesting. Isn't the lua-resty-string a part of the std library?

Reply:
Maybe, it's a blind spot in my knowledge, let me delete it.